services/viva-ms/lambdas/syncExpiryTime.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+import to from 'await-to-js';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+import config from '../../../config';
+import params from '../../../libs/params';
+
+import { getFutureTimestamp, millisecondsToSeconds } from '../../../libs/timestampHelper';
+import {
+  DELETE_VIVA_CASE_AFTER_45_DAYS,
+  DELETE_VIVA_CASE_AFTER_72_HOURS,
+} from '../../../libs/constants';
+
+const VIVA_CASE_SSM_PARAMS = params.read(config.cases.providers.viva.envsKeyName);
+
+export async function main(event) {
+  const { caseKeys } = event.detail;
+
+  const [getCaseError, getCaseResponse] = await to(getCase(caseKeys));
+  if (getCaseError) {
+    throw getCaseError;
+  }
+
+  const caseItem = getCaseResponse.Items[0];
+
+  const vivaCaseSSMParams = await VIVA_CASE_SSM_PARAMS;
+  if (vivaCaseSSMParams.recurringFormId !== caseItem.currentFormId) {
+    console.info('(Viva-ms: syncExpiryTime): case is not an recurring form. Nothing to update.');
```

Comment:
'(Viva-ms: syncExpiryTime): case is not an recurring form. Nothing to update.' -> '(Viva-ms: syncExpiryTime): case attribute currentFormId (caseItem.currentFormId) does not match the recurring form id (vivaCaseSSMParams.recurringFormId) from ssm params. Nothing to update.'

A bit more detailed information in my opinion :)

Reply:
to verbose in my opinion

Reply:
Then we need a second opinon. My take is that rather log a detailed description telling the reader what broke so that we know in detail why the lambda fail. In other words I rather have a more verbose logging to help me while debugging an issue :)